coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -161,7 +179,11 @@ func NewServer(
 	if options.AcquireJobLongPollDur == 0 {
 		options.AcquireJobLongPollDur = DefaultAcquireJobLongPollDur
 	}
-	return &server{
+	if options.HeartbeatInterval == 0 {
+		options.HeartbeatInterval = DefaultHeartbeatInterval
+	}
```

Comment:
Suggestion: use a default heartbeatfn to avoid the if nil check later on? `options.HeartbeatFn = func(context.Context) error { return s.Database... }`.

Alternatively, rename this `testHeartbeatFn` for clarity?

Reply:
The former 👍